frontend/src/components/main-view/details/details-card/DetailsCard.tsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react'
-import {images} from "../../../helpers";
-import {useAppSelector} from "../../../../../app/hooks";
-import {selectDetailsData} from "../../../../../slicer/detailsSlice";
+import {images} from "../../helpers";
+import {useAppSelector} from "../../../../app/hooks";
+import {selectDetailsData} from "../../../../slicer/detailsSlice";
```

Comment:
As Robert Martin would say, "Data" is not a type of clean coding. Maybe something like "selectDetailsCard".

Reply:
i guess selectDetails would be good enough